### DISCUSSION TITLE
What does a config do?

### DISCUSSION THREAD

**Discussion by ProjectileAimbotError404** [Fedoraware/Fedoraware#1179] [Q&A]
What does it do? And could you show examples

**Comment by johnathon-walnut** [comment]
It saves the current configuration of the features so you can load them again the same way in another session

configuration
/kənˌfɪɡəˈreɪʃn,kənˌfɪɡjʊˈreɪʃn/
noun
noun: configuration; plural noun: configurations

    1.
    an arrangement of parts or elements in a particular form, figure, or combination.
    "the unrepeatable configuration of the stars at the moment of your birth"

**Reply by ProjectileAimbotError404** [reply]
How do you set it up?

**Comment by BindUser** [comment]
You simply save it
